src/PrestaShopBundle/Form/Admin/Sell/Product/FooterType.php

```diff
@@ -97,15 +107,15 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('duplicate_product', IconButtonType::class, [
                 'label' => $this->trans('Duplicate', 'Admin.Actions'),
                 'type' => 'link',
-                'icon' => 'content_copy',
+                //'icon' => 'content_copy',
```

Comment:
As we discussed please leave the icons for now, we will discuss it with Tristan later, we didn't have an answer from him yet so no need to remove them for now

Reply:
commit dropped